resources/home/dnanexus/run_workflows/tests/test_manage_dict.py

```
@@ -1176,18 +1125,16 @@ def test_array_input_with_single_dict_given(self):
-            ],
+            input_classes=self.input_classes["workflow-GB12vxQ433GygFZK6pPF75q8"],
         )
 
         input_type = type(
             output["stage-G9Z2B8841bQY907z1ygq7K9x.somalier_extract_file"]
         )
 
-        assert (
-            input_type == list
-        ), "array:file input not converted to a list as expected"
+        assert input_type == list, (
```

Ruff (0.12.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)